ideascube/library/models.py

```diff
@@ -99,3 +101,10 @@ def __str__(self):
 
     def get_absolute_url(self):
         return reverse('library:book_detail', kwargs={'pk': self.item.pk})
+
+    @property
+    def extension(self):
+        if not self.file:
+            return ''
+        name, extension = os.path.splitext(self.file.name)
+        return extension[1:]
```

Comment:
for not so basic extension (like _.tar.gz_) it may not be reliable.

Maybe something like:
```
file_name = self.file.name
extension = file_name[file_name.find('.'):]
```